nuxt.config.js

```diff
@@ -35,9 +35,7 @@ export default {
 					})
 
 					callback(null, pageRoutes.concat(postRoutes))
-				}), function (err) {
-					return next(err)
-				})
+				}))
```

Comment:
@TeddTech I'm not sure this adequately addresses #27. By removing the error callback, this means that if the API is unavailable for any reason, the site will just break. Any thoughts on how you could handle this error?

Reply:
I can import express for this functionality?